docs/ref/release_notes.rst

```diff
@@ -0,0 +1,54 @@
+What's new in Iris-grib v0.13
+=============================
+
+:Release: 0.13.0
+:Date: T.B.D.
```

Comment:
I take it these (that is, `:Release:` and `:Date:`) have defined functionality in ReStructured Text?

Reply:
TBH I have no idea. 
I don't think they do anything special -- I just copied the cartopy one.

Reply:
Let's just go with it!